src/luascript.cpp

```diff
@@ -7327,19 +7327,32 @@ int LuaScriptInterface::luaCreatureGetPathTo(lua_State* L)
 
 int LuaScriptInterface::luaCreatureMove(lua_State* L)
 {
+	// creature:move(direction, tile[, flags = 0])
```

Comment:
I made it so that we would be forced to use ``direction`` even if we were targeting the form ``creature:move(tile[, flags = 0])``, so I guess I'll update it to welcome both signatures logic.

Please assist me if I don't do it right xD.